### PR TITLE
Physical keys cause app to crash solved.(Issue number 1928)

### PR DESCRIPTION
**Pull request by Paramjeet2810**
Physical key crash issue solved.

### CODE REVIEW

File: OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/MainActivity.java [hunk 1]
```diff
@@ -59,6 +60,7 @@
 
     public Drawer mDrawer;
     private Toolbar mToolbar;
+    private int CURSOR;
```
Comment (nmikhailov) [inline OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/MainActivity.java]: Check out coding style https://github.com/open-keychain/open-keychain#coding-style

**Comment by dschuermann** [comment]
Can you explain why this fixes the issue #1928 . What I see is that you implemented a form of navigation via physical buttons.

**Comment by Paramjeet2810** [comment]
Sir,
When I used the application with an OTG keyboard connected to my android
device, navigation through physical keys was no possible. Whenever I would
press any key the app wouldn't respond.
After adding the navigation via physical button it works perfectly fine.
After that I opened the app in an emulator which had a d-pad, the same code
worked even for the d-pad.

According to my understanding, the user had this problem that whenever he
used to use his physical keys to navigate in the Navigation Drawer as there
was no available functionality his app wouldn't function.
Regards,
Paramjeet Desai

**Comment by nmikhailov** [comment]
Have you managed to reproduce the crash though? Original issue #1928 clearly states that hardware keys crash OKC.

**Comment by Paramjeet2810** [comment]
No, I couldnt reproduce the crash exactly.

**Comment by Paramjeet2810** [comment]
I may not have reproduced the crash but I have solved the problem of app not responding when a keyboard is attached to the mobile. And if given the log-cat I would try to solve the crash problem if it still persists, but I am positive that the problem is solved.

**Comment by Paramjeet2810** [comment]
Actually I am not getting emulators of qwerty(with no touchscreen) devices. So as soon as I find one I will see to it that the app even works for those phones. But I am very positive that it will work as long as they follow the  universally accepted ASCII system.

**Comment by Valodim** [comment]
Hi! Sorry for the delay, I'm a little between things lately :(

Can you explain what the problem was? From what I understand in the code, you handle the onKeyUp method manually now. do you know why this crashed before?

**Comment by Paramjeet2810** [comment]
On key press event wasn't functioning so I had to manually add the code.
Try connecting a USB keyboard to your mobile and use the app. The app
doesn't work when the navigation drawer is open.

**Comment by Paramjeet2810** [comment]
Sir, can we have a video chat at sometime possible by you. I guess it would save alot of your time as well as mine. I want to contribute more in open-keychain and this could motivate me to work more efficiently.
@nmikhailov I am really sorry for taking such a long time for solving a small issue.

**Comment by dschuermann** [comment]
Sry, but we do not consider this an appropriate way to fix the problem.

**Pull request closed without merging**